README.md

```diff
@@ -31,9 +31,29 @@ trimCanvas(canvas)
 ```
 
 If you don't want to mess with your existing canvas, then simply clone the canvas element beforehand.
+`trim-canvas` v0.2+ has built-in support for cloning:
+
+```javascript
+const newTrimmedCanvas = trimCanvas(canvas, {clone: true})
+```
+
+Can view the [full list of options](#API) below.
 
 `trim-canvas` returns the canvas element for easy chaining.
 
+### API
+
+#### `trimCanvas(canvas, options)`
+
+- arguments
+  - **canvas** *Canvas object* The canvas to be trimmed.
+  - **options** *object* Optional arguments.
+    - **clone** *bool* Whether to clone the canvas before trimming (default: `false`).
+    - **createCanvas** *function* A custom function to create a Canvas (defaults to DOM implementation).
+      Supports [`node-canvas`'s `createCanvas`](https://github.com/Automattic/node-canvas#createcanvas).
+
+- returns the trimmed canvas (original or cloned)
+
 ## Example
 
 Can see how `trim-canvas` is used inside of `react-signature-canvas` [here](https://github.com/agilgur5/react-signature-canvas/blob/310bff81813509a4035bedfe50d76e7045a880cb/src/index.js#L53-L64).
```

Comment:
This link would be out-of-date with this release, as there's no need for `react-signature-canvas` to implement its own cloning logic after this release. Could link to an unreleased commit that uses the new cloning logic?